addons/mail/models/mail_message.py

```diff
@@ -1086,16 +1088,23 @@ def _cleanup_side_records(self):
         self.write({
             'starred_partner_ids': [(5, 0, 0)],
             'notification_ids': [(5, 0, 0)],
+            'pinned_on_channel_id': [Command.clear()]
         })
 
+    def _is_empty(self):
```

Comment:
Useless diff ? Does not seem used (anyway you may check the content of self versus self._filter_empty() to know if a message is empty.